src/app/components/job-card/job-card.component.scss

```diff
@@ -0,0 +1,61 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+$job-card-border-radius: 8px;
+
+.home-job-item-container {
+  display: inline-block;
+  width: 180px;
+  margin: 5px;
+  background-color: $default-bg-color;
+  border-radius: $job-card-border-radius;
+  box-shadow: 0 2px 6px 0 rgba(0,0,0,0.4);
+  cursor: pointer;
+
+  &:hover {
+    transform: scale(1.02);
+  }
+
+  .home-job-item-name-container {
+    height: 110px;
+    padding: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    .home-job-item-name {
+      text-align: center;
+      white-space: normal;
+      @include truncate-owerflowing-text;
+    }
+  }
+
+  .home-job-item-salary-container {
+    color: $white;
+    background: $color-primary-cerise;
+    padding: 10px;
+    font-weight: 500;
+    /* TODO: Jobs border radius variable */
+    border-radius: 0 0 $job-card-border-radius $job-card-border-radius;
+    text-align: center;
+  }
+
+  .home-job-item-salary-will-perform-job-container {
+    background: $color-primary-cerise;
+    color: $white;
+  }
+
+  .home-job-item-salary-job-ended-container {
+    background: $color-gray-50;
+    color: $color-gray-200;
+  }
+}
```

Comment:
Rule declaration should be followed by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)